catalog-info.yaml

```
@@ -0,0 +1,54 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: elastic-agent-system-metrics
+
+spec:
+  type: library
+  owner: group:ingest-fp
+  system: platform-ingest
+  lifecycle: production
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elastic-agent-inputs
+  description: Buildkite pipeline for the elastic-agent-inputs project
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/elastic-agent-inputs
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: elastic-agent-inputs
+      description: Buildkite pipeline for the elastic-agent-inputs project
+    spec:
+      repository: elastic/elastic-agent-inputs
```

missing branch_configuration. Just a tag `v0.0.0` created here in this repo... we could add a glob for that just in case

```suggestion
      branch_configuration: "main v0.*"
      repository: elastic/elastic-agent-inputs
```